examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -0,0 +1,71 @@
+"""
+==================================
+GOES XRS Near Real Time (NRT) data
```

Comment:
I would rephase this:

```suggestion
How to create a `sunpy.timeseries.TimeSeries` from the GOES-XRS near real time (NRT). 
```